js&css/web-accessible/www.youtube.com/appearance.js

```diff
@@ -157,10 +157,10 @@ ImprovedTube.formatSecond = function (rTime) {
 
 ImprovedTube.playerRemainingDuration = function () {
 	var duration = document.querySelector(".ytp-time-duration").innerText;
-	var remainingDuration = ImprovedTube.storage.player_remaining_duration;
-	if (remainingDuration) {
+//	var remainingDuration = ImprovedTube.storage.player_remaining_duration;
```

Comment:
Where did you comment? This is useful for the scenario that I mentioned in a previous comment.